src/components/FooterNavList/FooterNavList.tsx

```diff
@@ -0,0 +1,8 @@
+interface FooterNavListProps {
+  footerListItems: string[];
```

Comment:
`items` is fine, should remove prefix footer in prop since it already in Footer component